### PR TITLE
Create multi pillar tests

### PR DESCRIPTION
**Pull request by dupliaka**
Created the test that use change an swap pillar.
By any chance why move selector benchmarks almost never do steps?

### CODE REVIEW

File: optaplanner-benchmarks/README.md [hunk 1]
```diff
@@ -60,15 +60,17 @@ Testing planner's performance regression in various tests.
...
-   4. open the document and refresh all the tables and charts
-   5. provide analysis and comments on results
-   6. convert the report into PDF and share it on Mojo to make it searcheable
+## Performance profiling with JMH
+
+You can run the profiling on jmh by adding -prof flag with the path to the profiler
+
+
+For Example to create a flame graph with asyncprofiler 
+Download async profiler lib and unzip it
```
Comment (rsynek) [inline optaplanner-benchmarks/README.md:69]:
```suggestion
For Example to create a flame graph with the async profiler, download async profiler lib and unzip it:
```

File: optaplanner-benchmarks/README.md [hunk 2]
```diff
@@ -60,15 +60,17 @@ Testing planner's performance regression in various tests.
...
- - A new report can be created by following these steps:
-   1. copy both spreadsheet and document template into a new subfolder
-   2. upload CSV files with results for builds being compared from PerfRepo into the same subfolder
-   3. reference both files in the spreadsheet
-   4. open the document and refresh all the tables and charts
-   5. provide analysis and comments on results
-   6. convert the report into PDF and share it on Mojo to make it searcheable
+## Performance profiling with JMH
+
+You can run the profiling on jmh by adding -prof flag with the path to the profiler
```
Comment (rsynek) [inline optaplanner-benchmarks/README.md:65]:
```suggestion
You can run the profiling on jmh by adding -prof flag with the path to the profiler.
```

File: optaplanner-benchmarks/README.md [hunk 3]
```diff
@@ -60,15 +60,17 @@ Testing planner's performance regression in various tests.
...
+
+
+For Example to create a flame graph with asyncprofiler 
+Download async profiler lib and unzip it
+```
+wget https://github.com/jvm-profiling-tools/async-profiler/releases/download/v2.9/async-profiler-2.9-linux-x64.tar.gz
+tar -xvf async-profiler-2.9-linux-x64.tar.gz
+```
+Run tests with the profiler
```
Comment (rsynek) [inline optaplanner-benchmarks/README.md:74]:
```suggestion
Run tests with the profiler:
```

**Review by rsynek — APPROVED** [review]
Please see a couple of formatting suggestions.

Move selector benchmarks may take fewer steps because they are configured to use only a single move selector, which may not provide enough opportunities to improve the working solution (just my guess without any investigation).

**Comment by dupliaka** [comment]
Thanks @rsynek. If I want to check multiple move selectors that is cashed after every step I just configure AcceptedCountLimit. So I moved the union testing configuration

**Review by rsynek — APPROVED** [review]

**Pull request merged**